source/Government.cpp

```diff
@@ -115,29 +144,37 @@ void Government::Load(const DataNode &node)
 						grand.PrintTrace("Skipping unrecognized attribute:");
 				}
 		}
-		else if(child.Token(0) == "bribe" && child.Size() >= 2)
-			bribe = child.Value(1);
-		else if(child.Token(0) == "fine" && child.Size() >= 2)
-			fine = child.Value(1);
-		else if(child.Token(0) == "enforces" && child.HasChildren())
+		else if(key == "bribe" && hasValue)
+			bribe = add ? bribe + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(key == "fine" && hasValue)
+			fine = add ? fine + child.Value(valueIndex) : child.Value(valueIndex);
+		else if(add && key == "enforces" && child.HasChildren())
+		{
+			enforcementZones.clear();
+			enforcementZones.emplace_back(child);
+		}
+		else if(key == "enforces" && child.HasChildren())
+		{
+			enforcementZones.clear();
```

Comment:
You can handle the add and non-add case for `enforces` in the same block. (This makes `add enforces` not clear the list as my previous comment mentions.)
```suggestion
		else if(key == "enforces" && child.HasChildren())
		{
			if(!add)
				enforcementZones.clear();
```